RELEASES.md

```diff
@@ -0,0 +1,44 @@
+# CITGM Release Process
+
+This document describes the technical aspects of the citgm release process. The intended audience is those who have been authorized to create official releases for citgm, hosted on npm.
+
+## Who can make a release?
+
+Any member of the @nodejs/citgm team can make a release.
+
+## How to create a release
+
+1. Ensure your branch is up to date with upstream
+
+```bash
+$ git remote update -p
+$ git rebase upstream/master
+$ git diff upstream/master # this should be a no-op
+```
+
+2. Install semver sync
+
+```bash
+$ npm install -g semver-sync
+```
+
+3. Bump the version and create tag with semver-sync
+
+```bash
+$ semver-sync -t patch # for patch
+$ semver-sync -t minor # for minor
+$ semver-sync -t major # for major
```

Comment:
Is there a reason to use this and not just use `npm version patch|minor|major`? The preversion and postversion scripts look pretty useful, and it removes a dependency.

```json
"scripts": {
  "preversion": "npm test",
  "version": "npm run build && git add -A dist",
  "postversion": "git push && git push --tags && rm -rf build/temp"
}
```